src/components/BrewCard/BrewCard.jsx

```diff
@@ -0,0 +1,17 @@
+import './BrewCard.css';
+
+export default function BrewCard({ breweries }) {
```

Comment:
BrewCard suggests that this component might contain just one single brewery. BrewCards might be preferred.